integration_test/tests/wallet.rs

```diff
@@ -314,17 +314,37 @@ fn wallet__import_address() {
 #[cfg(not(feature = "v20_and_below"))]
 fn wallet__import_descriptors() {
     use node::{serde_json, ImportDescriptorsRequest};
+    use std::time::{SystemTime, UNIX_EPOCH};
 
     let node = Node::with_wallet(Wallet::None, &[]);
     let wallet_name = "desc_wallet";
     node.client.create_wallet_with_descriptors(wallet_name).expect("create descriptor wallet");
+    node.fund_wallet();
+
+    // 1. Get the current time
+    let start_time = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .expect("failed to get current time")
+        .as_secs();
 
+    // 2. Get an address and send some coins to it
     let address = node.client.new_address().expect("failed to get new address");
-    let descriptor = format!("addr({})", address);
+    let amount = bitcoin::Amount::from_sat(10_000);
+    let _txid = node.client.send_to_address(&address, amount).expect("sendtoaddress");
+
+    // 3. Get the descriptor for the address
+    let descriptor = node.client.get_descriptor_info(&format!("addr({})", address))
+        .expect("get_descriptor_info")
+        .descriptor;
+
+    // 4. Mine 100 blocks
+    let mining_address = node.client.new_address().expect("failed to get mining address");
+    let _blocks = node.client.generate_to_address(100, &mining_address).expect("generatetoaddress");
 
+    // 5. Scan for the descriptor using the time from (1)
     let request = ImportDescriptorsRequest {
         desc: descriptor,
-        timestamp: serde_json::Value::String("now".to_string()),
+        timestamp: serde_json::Value::Number(serde_json::Number::from(start_time)),
     };
 
     let _: ImportDescriptors = node.client.import_descriptors(&[request]).expect("importdescriptors");
```

Comment:
We want to check that the list isn't empty, right? Otherwise all the good work above is not validated.